src/form-host/cht-environment.js

```diff
@@ -1,35 +1,11 @@
-const $ = require('jquery');
 const { useFakeTimers } = require('sinon/lib/sinon/util/fake-timers');
-const { toBik_text } = require('bikram-sambat');
-const moment = require('moment');
 
-require('select2');
-
-// webapp/node_modules/bootstrap/js/dropdown.js expects this declared globally
-window.jQuery = $;
-// webapp/src/js/enketo/main.js expects this for datepicker
-window.$ = $;
-// backward compatibility with harness v2
-window.$$ = $;
+// // backward compatibility with harness v2
```

Comment:
```suggestion
// backward compatibility with harness v2
```

Reply:
since this is a breaking change, is now the time to remove this `$$` silliness? dunno what it is about exactly

Reply:
Agreed.  I have just removed `window.$$`